props/saber_BC_buttons.h

```diff
@@ -1278,6 +1558,12 @@ case EVENTID(BUTTON_POWER, EVENT_FOURTH_HELD, MODE_OFF):
   bool battle_mode_ = false;
   bool max_vol_reached_ = false;
   bool min_vol_reached_ = false;
+
+  bool main_blade_on_ = false;
```

Comment:
SaberBase actually keeps track of which blades are on, so we should be able to replace these bools with a function or macro that asks saberbase what is already on.

That will be easier if we make a class for it though.